services/mgmt/alertsmanagement/src/package_preview_2021_08/models.rs

```diff
@@ -5,56 +5,13 @@ use serde::{Deserialize, Serialize, Serializer};
 use std::str::FromStr;
 #[doc = "Action to be applied."]
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub struct Action {
-    #[doc = "Action that should be applied."]
-    #[serde(rename = "actionType")]
-    pub action_type: action::ActionType,
-}
+pub struct Action {}
```

Comment:
This doesn't feel correct.  I've checked a handful of these, and found each I've checked look to generate empty structs.

Reply:
I agree. I've removed the empty structs. It uncovered two other minor issues that I had to fix
- union types for vector aliases
- support for nested unions